### DISCUSSION TITLE
Welcome to Hacking-Tools Discussions!

### DISCUSSION THREAD

**Discussion by aw-junaid** [aw-junaid/Hacking-Tools#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by muhammedhack** [comment]
How hack Instagram

**Reply by aw-junaid** [reply]
Hacking tools are only made for penetration testing. Hacking Instagram mean's you are violating the Instagram privacy policy.

**Comment by CrYpToKiD14000** [comment]
do you know a software where i can apply codes on mac os like bassically when you have to script a site or make a script where do i go please help me :((((((((((((((((((((((

**Comment by aw-junaid** [comment]
For Mac Os Most Popular Script Editor is AppleScript.

**Comment by as777ms** [comment]
hi. I want to improve my IT and become a pentester. What firstly must I do? Can you advise to me something' somewhere to learn for free.

**Reply by aw-junaid** [reply]
Before penetration testing you must know about how Operating System works(**Operating System Concepts Book by Abraham Silberschatz**) and About Networking(**ANY CCNA GUIDE**)... In operating you must learn about **LINUX** and it's commands.
